src/arch/xtensa/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ elseif(CONFIG_IMX8M)
 	set(platform_folder imx8m)
 elseif(CONFIG_IMX8ULP)
 	set(platform_folder imx8ulp)
+elseif(CONFIG_MT8195)
+	set(platform_folder mt8195)
```

Comment:
Will MT8192 share no code at all with MT8195? If it will, how about renaming the common parts to something like `MT819x`?